### PR TITLE
Add brython@3.2.7 w/ npm auto-update config

### LINKED ISSUE #8312: [Request] Add brython
**Library name:** brython
**Git repository url:** https://github.com/brython-dev/brython
**npm package url(optional):** 
**License(s):** BSD
**Official homepage:** http://www.brython.info/
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
PR for #8312 
@kennynaoh @pvnr0082t  please help me review it, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required):https://github.com/brython-dev/brython
- Official website (optional, not the repository):
- NPM package url (optional):https://www.npmjs.com/package/brython
- GitHub / Bitbucket popularity (required):
  - Count of stars:915
  - Count of watchers:66
  - Count of forks:107
- NPM download stats (optional):
  - Downloads in the last day:2
  - Downloads in the last week:7
  - Downloads in the last month:55
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

close #8312, cc @PierreQuentel

### CODE REVIEW

**Comment by kennynaoh** [comment]
@x09326 
how about use npm auto-update?

File: ajax/libs/brython/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "keywords": [
+    "Python",
+    "browser"
+  ],
+  "author": "Pierre Quentel",
+  "license": "BSD-2-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brython-dev/brython.git",
+    "basePath": "",
```
Comment (kennynaoh) [inline ajax/libs/brython/package.json]: how about using `www/src`?

File: ajax/libs/brython/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "files": [
+      "www/src/brython.js",
+      "www/src/brython_dist.js"
```
Comment (kennynaoh) [inline ajax/libs/brython/package.json]: how about using `brython!(*+(builtins|loader)*)`?

File: ajax/libs/brython/package.json (hunk 1 above)
Comment (x09326) [inline ajax/libs/brython/package.json]: I think that is not a good basepath.

File: ajax/libs/brython/package.json (hunk 2 above)
Comment (x09326) [inline ajax/libs/brython/package.json]: Because the author said that we can just add the two files.

https://github.com/brython-dev/brython/issues/454

**Comment by x09326** [comment]
@kennynaoh @pvnr0082t 
Please help me review this PR.
I add old versions of this lib.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by PierreQuentel** [comment]
Hello,

The latest Brython version is 3.3.1. Releases have been simplified and now consist of only 2 scripts : __brython.js__ and __brython_stdlib.js__. Can you publish this version on cdnjs ?

**Comment by x09326** [comment]
@cdnjs/intern2 
So what I need to do is :
1. Add the latest version (3.3.1)
2. Manually add all old versions because the files structure are different from the latest version

Please give me some advices, thanks.

**Comment by pvnr0082t** [comment]
@x09326 I think it's okay. Go ahead.

**Comment by x09326** [comment]
@PierreQuentel 
Hi, can I just add the latest version of brython on npm?
Because the old file structure and file name are different from latest version on npm.

**Comment by PeterDaveHello** [comment]
@x09326 sounds great.

**Comment by PierreQuentel** [comment]
If you can wait for a few days, I will publish a version on npm with only 2 files. The current npm package has too many files.

**Comment by PeterDaveHello** [comment]
@PierreQuentel sounds very great, thanks a lot !

**Comment by x09326** [comment]
@PierreQuentel Thank you!

**Comment by PierreQuentel** [comment]
I have just published version 3.3.2 on npm.

**Comment by pvnr0082t** [comment]
@PierreQuentel Thanks.

**Comment by x09326** [comment]
@cdnjs/intern2 please help me review it, I have add the latest version.

File: ajax/libs/brython/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/brython-dev/brython.git"
+  },
+  "keywords": [
+    "Python",
+    "browser"
+  ],
+  "author": "Pierre Quentel",
+  "license": "BSD-2-Clause",
```
Comment (pvnr0082t) [inline ajax/libs/brython/package.json]: @x09326 It shows that it's `BSD-3-Clause`(https://github.com/brython-dev/brython/blob/3.2.7/LICENCE.txt) on GitHub repo, but it shows `BSD-2-Clause` on NPM. Please confirm it with author, thanks.

Comment (PierreQuentel) [inline ajax/libs/brython/package.json]: It was a typo, the licence is BSD-3. I have updated [package.json](https://github.com/brython-dev/brython/blob/master/npm/package.json) in the Brython repository

**Comment by x09326** [comment]
@pvnr0082t I have modified the license to BSD-3, please help me review it again, thanks.

File: ajax/libs/brython/package.json (hunk 3 above)
Comment (pvnr0082t) [inline ajax/libs/brython/package.json]: @PierreQuentel Thank you.

**Comment by PeterDaveHello** [comment]
This auto-update config will grab v3.2.6 with this structure, is this correct?
```
├── 3.2.6
│   ├── brython_builtins.js
│   ├── brython_dist.js
│   ├── brython.js
│   └── brython_loader.js
```

**Comment by PierreQuentel** [comment]
The files should be just brython.js and brython_stdlib.js. Versions before 3.3.0 didn't include brython_stdlib.js and should not be published.

**Comment by PeterDaveHello** [comment]
@PierreQuentel thanks!

**Comment by x09326** [comment]
@PeterDaveHello So what I need to do is modify the `.gitignore` or just modify the auto-update config?

**Comment by PeterDaveHello** [comment]
@x09326 Use .gitignore please

**Pull request closed without merging**